src/theme/BlogArchivePage/index.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
```

Comment:
The original theme files are in TypeScript. Can we also use TypeScript for all the blog customization templates?

https://github.com/facebook/docusaurus/blob/main/packages/docusaurus-theme-classic/src/theme/BlogArchivePage/index.tsx